cloud/blockstore/libs/storage/partition/part_actor_readblocks.cpp

```diff
@@ -1059,10 +1059,15 @@ void TPartitionActor::CompleteReadBlocks(
     RemoveTransaction(*args.RequestInfo);
 
     if (args.Interrupted) {
+        ui32 flags = 0;
+        SetProtoFlag(flags, NProto::EF_SILENT);
```

Comment:
Here we also mute errors https://github.com/ydb-platform/nbs/blob/main/cloud/blockstore/libs/storage/partition/part_actor_readblocks.cpp#L704 and https://github.com/ydb-platform/nbs/blob/main/cloud/blockstore/libs/storage/partition/part_actor_readblocks.cpp#L747.

I believe it is OK, because those errors happen when `TGuardedSgList` is empty which and that means that corresponding request is already canceled